### ISSUE TITLE
Bridge 1.14 fails to compile on Mono

### ISSUE THREAD

**Issue by kcudnik** [bridgedotnet/Bridge#1482]
Issue is located in Bridge/Html5/Bridge.Html5.csproj
since linux treat file names as case sensitive, line

```html
<Compile Include="Elements\HTMLElement.cs" />
```

should be:

```html
<Compile Include="Elements\HtmlElement.cs" />
```

### Expected

no errors

### Actual

```cs
/tmp/Bridge/Html5/CSC : error CS2001: Source file `Elements/HTMLElement.cs' could not be found
```

### Steps To Reproduce

```cs
mdtool build Bridge.sln 
```

**Comment by LeonidVeriga** [comment]
Thank you for reporting, @kcudnik. We will try to fix this in 1.15.

**Comment by LeonidVeriga** [comment]
Will be fixed along with #1719

**Comment by LeonidVeriga** [comment]
Hi @kcudnik,

We fixed the issue.

And now, in addition to CI build on MS .Net ([App Veyor](https://ci.appveyor.com/project/ObjectDotNet/bridge/build/15.0.762-master)), we have CI build on Mono ([Travis](https://travis-ci.org/bridgedotnet/Bridge/builds/157518858)).
